lib/app/features/chat/messages/views/components/message_types/money_receive_message.dart

```diff
@@ -0,0 +1,114 @@
+// SPDX-License-Identifier: ice License 1.0
+
+import 'package:flutter/material.dart';
+import 'package:ion/app/components/button/button.dart';
+import 'package:ion/app/extensions/extensions.dart';
+import 'package:ion/app/features/chat/messages/views/components/mesage_timestamp.dart';
+import 'package:ion/app/features/chat/messages/views/components/message_item_wrapper.dart';
+import 'package:ion/generated/assets.gen.dart';
+
+class MoneyReceiveMessage extends StatelessWidget {
+  const MoneyReceiveMessage({required this.isMe, super.key});
+  final bool isMe;
+
+  @override
+  Widget build(BuildContext context) {
+    return MessageItemWrapper(
+      contentPadding: EdgeInsets.symmetric(
+        horizontal: 12.0.s,
+        vertical: 12.0.s,
+      ),
+      isMe: isMe,
+      child: Column(
```

Comment:
Could you please split this widget tree into smaller classes for better readability?